runtime/src/common.rs

```diff
@@ -0,0 +1,50 @@
+use codec::{Encode};
+use rstd::vec::Vec;
+
+// default substrate child storage root
+const CHILD_STORAGE_DEFAULT_PREFIX: &[u8] = b":child_storage:default:";
+
+const MS_PER_YEAR: u64 = 86400000;
+
+/// get days(round up) since epoch given the timestamp in ms
+pub fn get_days_since_epoch(ts: u64) -> u32 {
+    let remainder = ts % MS_PER_YEAR;
+    let days = (ts / MS_PER_YEAR) as u32;
+    if remainder == 0 {
+        days
+    } else {
+        days + 1
+    }
+}
+
+/// create a child storage key from the given specific key
+pub fn child_storage_key(specific_key: u32) -> Vec<u8> {
```

Comment:
would also call this `get_xxx` or `generate_xxx` to be clear what the method does.